portal-web/docroot/html/taglib/ui/icon/page.jsp

```diff
@@ -23,7 +23,7 @@ String listItemAnchorAriaHasPopup = "false";
 String listItemAnchorAriaRole = "menuitem";
 String listItemAriaRole = "presentation";
 
-if (useDialog || (urlIsNotNull && url.startsWith("javascript:"))) {
+if (!linkCssClass.contains("keep-aria-attributes") && (useDialog || (urlIsNotNull && url.startsWith("javascript:")))) {
```

Comment:
I took a deeper look at this one (Carlos and I talked about this in a call), because it's a bit odd that we "pass" data via css-classes that determine other behavior. I was in doubt whether doing this with an explicit API instead, however, just saw that the code we're changing here was added recently to solve [other a11y issue](https://github.com/liferay/liferay-portal/commit/e5e8d89fc66dcd972277111b9f75304e8cf7a8b4#diff-ca7f892e372c1b8f96759da54347b28278d58c983406f3461818d7e07062e1caR21). 

So, basically, that other fix did not account for the case of inpuy-localized.

We may pass this but I'd like to make sure there's no other reasonable way @markocikos  any idea?